setup.py

```diff
@@ -12,14 +14,35 @@
     version = 'dev'
 
 
-if __name__ == '__main__':
-    setup(name='conda-smithy',
-          version='0.1.0dev0',
-          description='A package to create repositories for conda recipes, and automate their building with CI tools on Linux, OSX and Windows.',
-          author='Phil Elson',
-          author_email='pelson.pub@gmail.com',
-          url='https://github.com/conda-forge/conda-smithy',
-          scripts=[os.path.join('scripts', 'conda-smithy')],
-          packages=['conda_smithy'],
-         )
+def main():
+    skw = dict(
+        name='conda-smithy',
+        version='0.1.0dev0',
+        description='A package to create repositories for conda recipes, and automate '
+                    'their building with CI tools on Linux, OSX and Windows.',
+        author='Phil Elson',
+        author_email='pelson.pub@gmail.com',
+        url='https://github.com/conda-forge/conda-smithy',
+        scripts=[os.path.join('scripts', 'conda-smithy')],
```

Comment:
OK, merging this, since the change to an entry point will need some file re-arranging too. And it isn't as immediate as the ability to install on 'nix-like systems.